scheduler/src/cook/mesos/api.clj

```diff
@@ -2808,6 +2932,23 @@
              :get {:summary "Returns the pools."
                    :handler (pools-handler)}})))
 
+      (c-api/context
+       "/data-local/:uuid" []
+       :path-params [uuid :- s/Uuid]
+       (c-api/resource
+        {:produces ["application/json"]
+         :responses {200 {:schema DataLocalCostResponse}}
+         :get {:summary "Returns summary information on the current data locality status"
+               :handler (data-local-cost-handler conn)}}))
+
+      (c-api/context
+       "/data-local" []
```

Comment:
Worth a comment here that this code is intended for diagnostic purposes and is not expected to be performant.

Reply:
Done